CHANGELOG

```diff
@@ -0,0 +1,35 @@
+v1.0.3
```

Comment:
Adding a CHANGELOG file to track changes, we may raw include it later on in documentation if needed.